configure

```diff
@@ -856,7 +862,7 @@ datadir=$datadir
 
 Name: bitlbee
 Description: IRC to IM gateway
-Requires: glib-2.0
+Requires: $pkgconfigrequires
```

Comment:
Why do we need this ? AFAIK bitlbee's public API doesn't include json.h, and we don't install static libraries?

Reply:
`json_util.h` includes `json.h` directly which is part of public API. Actaully, `json_util.h` is used in `bitlbee-steam` in `steam-json.h` which suffers with
```
<root> steam - Login error: Parser: Memory allocation failure
```
when `external-json-parser` is used.

Reply:
Hmm, looking at the Makefile we currently install all headers. We should probably limit it to the specific things that e.g. plugins need.

Reply:
Yes, I agree, but in some other PR in my opinion. Anyway, `json_util.h` is currently required by `bitlbee-steam` plugin, so, this one should be probably preserved. It uses types from `json.h` in function prototypes, which somewhat supports this new requirement :)

Reply:
Okay, we should definitely discourage them from using it. But let's leave that to a separate PR.